lang/java/maven-plugin/src/main/java/org/apache/avro/mojo/AbstractAvroMojo.java

```diff
@@ -164,6 +164,15 @@ public abstract class AbstractAvroMojo extends AbstractMojo {
    */
   protected boolean createSetters;
 
+  /**
+   * The createNullSafeAnnotations parameters adds @Nullable and @NotNull
+   * annotations for fhe fields of the record. The default is to not include
+   * annotations.
```

Comment:
Can we please mention here that the annotations are JetBrains annotations?

Then it'll automatically be documented by the plugin help. Ideally, it also lists the dependency coordinates / package-url (`pkg:maven/org.jetbrains/annotations@24.0.1`).

Reply:
At this point in time, there are no other places on the website that document the maven plugin, or code generation options. It's only mentioned in passing on the "Getting started" page for Java.